pyfr/backends/opencl/types.py

```diff
@@ -7,7 +7,13 @@
 from pyfr.util import lazyprop
 
 
-class OpenCLMatrixBase(base.MatrixBase):
+class _OpenCLMatrixCommon(object):
+    @property
+    def _as_parameter_(self):
+        return self.data
```

Comment:
We don't seem to have .int_ptr here.  @nloppi Thoughts?  Does this cause any problems?

Reply:
Yes, it is required by CLBlast.